Pipfile.lock

```diff
@@ -18,18 +18,18 @@
     "default": {
```

Comment:
The Pipfile itself didn't change, so it looks like these changes are unrelated to the main work of this PR.

Reply:
@LindsayYoung Let's back out changes to the Pipfile.lock since they're unrelated, does that sound good to you?

Reply:
I wanted to add the awscli as a dev dependency, but I did not realize it was already a dev dependency,(Thanks past me), and I just updated them when I re-added the packages. 

I would like to merge the lock because the updated packages are what you and I deployed.

But, thank you for catching this.

Reply:
OK, that makes sense to me!